### DISCUSSION TITLE
When I exit the software, it can prompt me whether to save the llc project file

### DISCUSSION THREAD

**Discussion by syayys** [mifi/lossless-cut#1689] [Ideas]
When I exit the software, it can prompt me whether to save the llc project file。
thanks

**Comment by mifi** [comment]
LosslesCut currently saves llc project file automatically, do you not want that? I think a lot of people would not want to answer a dialog/question every time they close/open a new file.

**Reply by hrxn** [reply]
Has the option to turn off automatically saving the llc project file been removed?

**Reply by mifi** [reply]
no. it's called "Auto save project file"

**Reply by hrxn** [reply]
Ah, it's hidden under "Advanced Options" now, I see..

**Reply by syayys** [reply]
sorry,sometimes, i don't need the llc project file，it just may be a try. Too many files with the same name feels a bit confusing，Thank you for your dedication
